src/ui/AP2DataPlot2D.cpp

```diff
@@ -1074,7 +1053,7 @@ void AP2DataPlot2D::itemEnabled(QString name)
         }
         // use golden ratio for evenly distributed colors
         double golden_ratio_conjugate = 0.618033988749895;
-        double h = ((double)rand() / (double)(RAND_MAX));
+        double h = (static_cast<double>(rand()) / static_cast<double>(RAND_MAX));
```

Comment:
casting only one side seems to be enough but using double(vale) cast leads to a clang warning about "old style cast". So I would like to stay with the static_cast<>() style.